docs/Dataplex-batchsink.md

```diff
@@ -2,136 +2,141 @@
+Google Cloud Platform Console. This is the project that the Dataplex job will run in. If a temporary bucket needs to be
+created, the service account must have permission in this project to create buckets.
 
 **Service Account**  - service account key used for authorization
 
-* **File Path**: Path on the local file system of the service account key used for
-  authorization. Can be set to 'auto-detect' when running on a Dataproc cluster.
-  When running on other clusters, the file must be present on every node in the cluster.
+* **File Path**: Path on the local file system of the service account key used for authorization. Can be set to '
+  auto-detect' when running on a Dataproc cluster. When running on other clusters, the file must be present on every
+  node in the cluster.
 
 * **JSON**: Contents of the service account JSON file.
 
-**Location ID**: Location ID in which the Dataplex lake has been created. On the **Lake** page, click the name of the lake.
-The lake page has the location ID.
+**Location ID**: ID of the location in which the Dataplex lake has been created, which can be found on the details page
+of the lake.
 
-**Lake ID**: ID for the Dataplex lake. You can find it on the **Lake** detail page in Dataplex.
+**Lake ID**: ID of the Dataplex lake, which can be found on the details page of the lake.
 
-**Zone ID**: ID for the Dataplex zone. You can find it on the **Zone** detail page in Dataplex.
+**Zone ID**: ID of the Dataplex zone, which can be found on the details page of the zone.
 
-**Asset ID**: ID for the Dataplex Asset. It represents a cloud resource that is being managed within a lake as a member
+**Asset ID**: ID of the Dataplex Asset. It represents a cloud resource that is being managed within a lake as a member
 of a zone.
 
 **Asset Type**: Type of asset selected to ingest the data in Dataplex.
-* Bigquery Dataset - Asset is of type BigQuery dataset.
+
+* BigQuery Dataset - Asset is of type BigQuery dataset.
 * Storage Bucket - Asset is of type Storage Bucket.
 
 BigQuery Dataset specific fields
 ----------
 
-**Table Name**: Table to write to. A table contains individual records organized in rows.
-Each record is composed of columns (also called fields).
-Every table is defined by a schema that describes the column names, data types, and other information.
+**Table Name**: Table to write to. A table contains individual records organized in rows. Each record is composed of
+columns (also called fields). Every table is defined by a schema that describes the column names, data types, and other
+information.
 
 **Operation**: Type of write operation to perform. This can be set to Insert, Update or Upsert.
+
 * Insert - all records will be inserted in destination table.
-* Update - records that match on Table Key will be updated in the table. Records that do not match
-  will be dropped.
+* Update - records that match on Table Key will be updated in the table. Records that do not match will be dropped.
```

Comment:
change the dash to a period and change records to Records

Reply:
done